test/unit/lib/getMergedPullRequestsSpec.js

```diff
@@ -120,4 +120,18 @@ describe('getMergedPullRequests', function () {
         return expect(getMergedPullRequests(anyRepo))
             .to.become(expectedResults);
     });
+
+    it('should skip with non-matching parenthesis', function () {
+        const gitLogMessages = [
+            'Merge pull request #3 from kadirahq/greenkeeper-update-all'
```

Comment:
How would this case be possible? The parenthesis are always added by us in the custom `git log` format.